internal/tiltfile/tiltfile_test.go

```diff
@@ -896,6 +897,21 @@ docker_build("gcr.io/foo", "foo", cache='/path/to/cache')
 	f.loadErrString("no such file or directory")
 }
 
+func TestFilterYaml(t *testing.T) {
+	f := newFixture(t)
+	defer f.TearDown()
+	f.file("k8s.yaml", yaml.ConcatYAML(testyaml.DoggosDeploymentYaml, testyaml.SnackYaml, testyaml.SanchoYAML))
+	f.file("Tiltfile", `labels = {'app': 'doggos'}
```

Comment:
[Label selectors in Kubernetes](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors) offer other options than just strict equality, such as:
* Not equal
* Set contains

If people would like that supported in the future I don't think that could be done in the current syntax. What do you think about changing this

```python
labels = {'app': 'doggos'}
```
in to this:
```python
labels = ["app=doggos"]
```

So that we could do things like this in the future?
```python
labels = ["app=doggos", "environment!=prod"]
```

Reply:
I think we should support dict's as well, can easily enough make the type fuzzy and support strings later.